### PR TITLE
Label workflow

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Moving this from yogbot to github actions will probably be more stable, and reduces the amount of interaction with the github API yogbot needs to do. I'll figure out how to make it read the PR and apply it based on changelog PR content as well

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Changes to a .dmm
+Map Change: '**/*.dmm'
+
+# Any changes to .ogg files are marked as sound
+Sound: '**/*.ogg'
+
+# Changes to the SQL subfolder
+SQL: 'SQL/**'
+
+# Encompasses both the tgui and the tgui-net folder
```
Comment (alexkar598) [inline .github/labeler.yml]: Tgui-net :thonk:

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+tgui: '{tgui,tgui-next/**}'
```
Comment (alexkar598) [inline .github/labeler.yml:25]: Maybe rename it to UI and add css and HTML files too

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Use this to test the matches: http://www.globtester.com/
+
+# Any file within the config subfolder
+Config: 'config/**'
+
+# When the .DME is changed
+DME Edit: '**/*.dme'
```
Comment (alexkar598) [inline .github/labeler.yml:7]: Do we really need this tag

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: .github/labeler.yml (hunk 3 above)
Comment (Amelia0010) [reply .github/labeler.yml:7]: It's there already, I only added the Github label.

File: .github/labeler.yml (hunk 2 above)
Comment (Amelia0010) [reply .github/labeler.yml:25]: but why

**Pull request merged**